app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WidgetManagerDelegate.java

```diff
@@ -32,6 +32,7 @@
     void removeUpdateListener(@NonNull UpdateListener aUpdateListener);
     void pushBackHandler(@NonNull Runnable aRunnable);
     void popBackHandler(@NonNull Runnable aRunnable);
+    void pushIsServo(boolean aIsServo);
```

Comment:
I'd rename this to something else like `setIsServo` or `setIsServoSession`. Using `push` prefix normally indicates that there is also a `pop`.